java/client/src/org/openqa/selenium/remote/DriverCommand.java

```diff
@@ -318,4 +318,11 @@ static CommandPayload SET_CURRENT_WINDOW_SIZE(Dimension targetSize) {
   // Mobile API
   String GET_NETWORK_CONNECTION = "getNetworkConnection";
   String SET_NETWORK_CONNECTION = "setNetworkConnection";
+
+  // Cast Media Router API
+  String GET_CAST_SINKS = "getCastSinks";
```

Comment:
These command names are specific to Chromium-based browsers. Please move to `ChromiumDriverCommand`